### PR TITLE
Update PPP

### PR DESCRIPTION
**Pull request by ChongCheeKaiClarence**

### CODE REVIEW

File: docs/team/chongcheekaiclarence.md [hunk 1]
```diff
@@ -10,23 +10,38 @@ SoConnect is a desktop app for NUS SoC students to manage their contacts, optimi
...
-
-* **Project management**:
-    * Set up Github project board.
-
-* **Enhancements to existing features**:
-  to be added soon
-
-* **Documentation**:
+* **Enhancements implemented**:
+* **Tag Create Feature**: Added the ability to create a tag to the Tag List.
```
Comment (PangKuangWei) [inline docs/team/chongcheekaiclarence.md]: To standardise, add an indentation level here and `tag` and `Tag` is not consistent.

File: docs/team/chongcheekaiclarence.md [hunk 2]
```diff
@@ -10,23 +10,38 @@ SoConnect is a desktop app for NUS SoC students to manage their contacts, optimi
...
+    * What it does: allows the user to create a tag in the tagList so it can be used to tag contacts and todos.
+    * Justification: This feature allows categorisation of contacts and tasks, so it is easier to search for things related to the tag.
+    * Highlights: This enhancement affects `tag add`, `tag delete` and `tag edit` commands. It required an in-depth analysis of design alternatives. The implementation too was challenging as it required changes to `add` and `edit` commands.
```
Comment (PangKuangWei) [inline docs/team/chongcheekaiclarence.md]: Perhaps remove the word `too`?

**Review by PangKuangWei — COMMENTED** [review]
Some small suggestions, otherwise LGTM.

File: docs/team/chongcheekaiclarence.md (hunk 2 above)
Comment (ChongCheeKaiClarence) [reply docs/team/chongcheekaiclarence.md]: Ok. I'll remove it

File: docs/team/chongcheekaiclarence.md (hunk 1 above)
Comment (ChongCheeKaiClarence) [reply docs/team/chongcheekaiclarence.md]: Alright, then I will use `tag` and `tagList`.

**Pull request merged**